test/drawers/drawerTests.ts

```diff
@@ -20,10 +20,12 @@ class MockAnimator implements Plottable.Animators.Plot {
   }
 }
 
-class MockDrawer extends Plottable.Drawer {
-  public _drawStep(step: Plottable.Drawers.AppliedDrawStep) {
-    step.animator.animate(this.renderArea(), step.attrToAppliedProjector);
-  }
+function getMockDrawer(dataset: Plottable.Dataset) {
```

Comment:
Rename function to `constructMockDrawer()` or `createMockDrawer()`?

Also, unless we are having this be the endpoint for having a mock drawer, we should probably name this function to be more aligned with what you are mocking.